### PR TITLE
this adds a live ebuild (e.g. it's built from master) package

### PR DESCRIPTION
**Pull request by Phate6660**
This is especially useful for someone like who would like to get the 
latest xmake from the master branch of the git repo upon installing the package.

### CODE REVIEW

File: dev-util/xmake/xmake-9999.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+DESCRIPTION="A cross-platform build utility based on Lua."
+HOMEPAGE="https://xmake.io"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/xmake-io/xmake"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-vcs/git virtual/pkg-config"
```
Comment (waruqi) [inline dev-util/xmake/xmake-9999.ebuild:17]: Will it pull all submodules from git automatically?

File: dev-util/xmake/xmake-9999.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit git-r3
+
+DESCRIPTION="A cross-platform build utility based on Lua."
+HOMEPAGE="https://xmake.io"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/xmake-io/xmake"
```
Comment (waruqi) [inline dev-util/xmake/xmake-9999.ebuild:11]: Will it pull all submodules from git automatically?

**Review by Phate6660 — COMMENTED** [review]
Yep, it'll automatically pull all submodules.
Don't worry, I make sure to test before I push changes. :)
 
Edit: I swear I messaged in the reply box to the review, not sure how this happened.

**Comment by waruqi** [comment]
Ok, Thanks.

**Pull request merged**